app/components/organisms/HeaderWithSearch.tsx

```diff
@@ -8,8 +9,12 @@ export default function HeaderWithSearch() {
 	const searchBar = isOnSearchPage ? null : <SearchBar status="idle" />
 
 	return (
-		<header className="bg-primary/10 py-6 dark:bg-dark-primary/10">
+		<header className="dark:bg-dark-primary/10 bg-primary/10 py-6">
 			<nav className="container flex flex-wrap items-center justify-between gap-4 sm:flex-nowrap md:gap-8">
+				<div>
+					<img src={logo} alt="Epic News Logo" className="w-16" />
```

Comment:
The logo looks a bit small. Can you enlarge it, and make it fully circular? You can use the [`rounded-full`](https://tailwindcss.com/docs/border-radius) class to make it circular in shape.